pkg/util/util.go

```diff
@@ -50,3 +57,31 @@ func EqualStringSlice(a, b []string) bool {
 	}
 	return true
 }
+
+func GetCmdLine(pid int) (string, error) {
+	data, err := ioutil.ReadFile(fmt.Sprintf("/proc/%d/cmdline", pid))
+	if err != nil {
+		return "", err
+	}
+	result := string(bytes.ReplaceAll(data, []byte("\x00"), []byte(" ")))
+	return result, err
```

Comment:
more idiomatic and clearer to return result, nil

Reply:
done